harness/determined/exec/pid_client.py

```diff
@@ -7,7 +7,7 @@
     parser = argparse.ArgumentParser()
     parser.add_argument("addr")
     parser.add_argument("cmd")
-    parser.add_argument("cmd_args", nargs="*")
+    parser.add_argument("cmd_args", nargs=argparse.REMAINDER)
```

Comment:
Are you sure this is going to work right?  In my former experiments, the normal_arg + REMAINDER behaved strangely, but using just REMAINDER to mean command+cmd_args worked as intended.

Reply:
just ran a script with `localhost -- python3 -m determined.exec.worker_process_wrapper RANK --`, and got the expected output of (addr='localhost', cmd='python3', cmd_args=['-m', 'determined.exec.worker_process_wrapper', 'RANK', '--'])